.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,192 @@
+name: CI Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+    - develop
+
+# Cancel previous jobs if an update has been made to the pull request
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/develop' }}
+
+env:
+  CLANG_DOCKER_IMAGE: axom/tpls:clang-14_04-01-25_17h-34m
+  GCC_DOCKER_IMAGE: axom/tpls:gcc-13_04-01-25_17h-35m
```

Comment:
Do we need to update some sphinx docs to refer to this file instead of the azure stuff?

Reply:
/gaslight

I don't know what you are talking about @kennyweiss . The docs are already updated and perfect.

/gaslight

Reply:
Is that one of the automated github commands that you added :)